tesseract/tesseract_planning/tesseract_planners/cmake/tesseract_planners-config.cmake.in

```diff
@@ -5,9 +5,8 @@ set_and_check(@PROJECT_NAME@_INCLUDE_DIRS "${PACKAGE_PREFIX_DIR}/include")
 set_and_check(@PROJECT_NAME@_LIBRARY_DIRS "${PACKAGE_PREFIX_DIR}/lib")
 
 include(CMakeFindDependencyMacro)
-find_dependency(ompl)
-find_dependency(trajopt)
-find_dependency(tesseract_environment)
-find_dependency(tesseract_kinematics)
+find_dependency(tesseract_common)
+find_dependency(tesseract_geometry)
+find_dependency(tesseract_collision)
 
```

Comment:
Did you intend to change this file? I noticed that dependency ompl and trajopt was removed?